### PR TITLE
Fix docs and tidy README

### LINKED ISSUE #259: The docs are now worse?
I can't tell from the existing issues if there is visibility on this, but the docs are now much worse than before we changed the template.

Templates are visible, things are missing.
image

This is worse to navigate.
image

The dev instructions have gone?

### PR DESCRIPTION
**Pull request by coretl**
Fixes #259

### CODE REVIEW

**Review by coretl — COMMENTED** [review]
Docs are artifacts of the build job, and can be downloaded and reviewed locally here: https://github.com/bluesky/ophyd-async/actions/runs/8898088019/artifacts/1461193368

**Comment by abbiemery** [comment]
> Docs are artifacts of the build job, and can be downloaded and reviewed locally here: https://github.com/bluesky/ophyd-async/actions/runs/8898088019/artifacts/1461193368

This is not a great way to view this, can't I view on GH pages for this branch?

File: README.md
Comment (abbiemery) [inline README.md:1]: "This is a python library" or "Ophyd async is a python..." for the line after the box of links. It is okay for the first line under the title to be a tag line and not correct English, but The rest of the read me should be.

**Review by abbiemery — CHANGES_REQUESTED** [review]
I'd change the first line but other than that it seams to have fixed the formatting issues.

**Comment by coretl** [comment]
> > Docs are artifacts of the build job, and can be downloaded and reviewed locally here: https://github.com/bluesky/ophyd-async/actions/runs/8898088019/artifacts/1461193368
> 
> This is not a great way to view this, can't I view on GH pages for this branch?

No, pushing to gh-pages is what was bloating our repo size, so we removed it in the copier update. There's a possibility we could do this as part of https://github.com/DiamondLightSource/python-copier-template/issues/35

**Review by evalott100 — APPROVED** [review]

**Pull request merged**